### ISSUE TITLE
Error messages should have a higher color contrast ratio

### ISSUE THREAD

**Issue by joshkimux** [AIGAdc/dcdesignweek.org#420]
![image](https://netlify-cocoon.netlify.app/.netlify/functions/fetch?code=307&path=eyJzaXRlX2lkIjoiMTRiNDc5ZjMtZWM2YS00ZDBiLWI5NGUtNmZlZmUzMzc4NWUzIiwiZGVwbG95X2lkIjoiNjExZTg5OGE5NDQ4N2IwMDA3NTUxNTAzIiwiaWQiOiIxZDExMTc2Zi02NjljLTQwZGEtODVlYi1kZTZjMWQwZDYyZGYifQ==)

### Description of issue
Error message contrast will fail to meet a11y contrast needs for a number of colorblind situations. Consider an alternative format e.g. yellow/black or white/black to solve this.

![image](https://netlify-cocoon.netlify.app/.netlify/functions/fetch?code=307&path=eyJzaXRlX2lkIjoiMTRiNDc5ZjMtZWM2YS00ZDBiLWI5NGUtNmZlZmUzMzc4NWUzIiwiZGVwbG95X2lkIjoiNjExZTg5OGE5NDQ4N2IwMDA3NTUxNTAzIiwiaWQiOiI3Mjc4NGRiYy1hOTgzLTRiODctOWM0Yy04ZWZiMzNlNWJjNzIifQ==)

```
Path:      /
Browser:   Chrome 92.0.4515.131 on Mac OS 10.15.7
Viewport:  1277 x 1009 @2x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-359--dcdesignweek.netlify.app%2F&os=OS%20X&os_version=Catalina&browser=Chrome&browser_version=92.0&resolution=1277x1009&speed=1&start=true&ref=netlify-source)

Submitted from [Netlify Deploy Preview #359](https://deploy-preview-359--dcdesignweek.netlify.app/)

**Comment by marcusrelacion** [comment]
@joshkimux this has been fixed.
